src/celengine/globular.h

```diff
@@ -52,42 +52,42 @@ class Globular : public DeepSkyObject
     float getHalfMassRadius() const;
     unsigned int cSlot(float) const;
 
-    virtual float getBoundingSphereRadius() const { return tidalRadius; }
-
-    virtual bool pick(const celmath::Ray3d& ray,
-                      double& distanceToPicker,
-                      double& cosAngleToBoundCenter) const;
-    virtual bool load(AssociativeArray*, const std::string&);
-    virtual void render(const Eigen::Vector3f& offset,
-                        const Eigen::Quaternionf& viewerOrientation,
-                        float brightness,
-                        float pixelSize,
-                        const Renderer* r = nullptr);
-    virtual void renderGlobularPointSprites(const Eigen::Vector3f& offset,
-                                            const Eigen::Quaternionf& viewerOrientation,
-                                            float brightness,
-                                            float pixelSize);
+    float getBoundingSphereRadius() const override { return tidalRadius; }
+
+    bool pick(const celmath::Ray3d& ray,
+              double& distanceToPicker,
+              double& cosAngleToBoundCenter) const override;
+    bool load(AssociativeArray*, const std::string&) override;
+    void render(const Eigen::Vector3f& offset,
+                const Eigen::Quaternionf& viewerOrientation,
+                float brightness,
+                float pixelSize,
+                const Renderer* r = nullptr) override;
+
     GlobularForm* getForm() const;
 
-    virtual uint64_t getRenderMask() const;
-    virtual unsigned int getLabelMask() const;
-    virtual const char* getObjTypeName() const;
+    uint64_t getRenderMask() const override;
+    unsigned int getLabelMask() const override;
+    const char* getObjTypeName() const override;
 
  private:
+    void renderGlobularPointSprites(const Eigen::Vector3f& offset,
+                                    const Eigen::Quaternionf& viewerOrientation,
+                                    float brightness,
+                                    float pixelSize);
+
    // Reference values ( = data base averages) of core radius, King concentration
    // and mu25 isophote radius:
-
     static constexpr float R_c_ref = 0.83f, C_ref = 2.1f, R_mu25 = 40.32f;
 
     void recomputeTidalRadius();
 
- private:
-    float detail{ 1.0f };
-    std::string* customTmpName{ nullptr };
+    float         detail{ 1.0f };
+    std::string*  customTmpName{ nullptr };
     GlobularForm* form{ nullptr };
-    float r_c{ R_c_ref };
-    float c{ C_ref };
-    float tidalRadius{ 0.0f };
+    float         r_c{ R_c_ref };
```

Comment:
Why these 3 whitespaces?

Reply:
I prefer names of variables are aligned.